Model/Gateway.php

```diff
@@ -387,6 +388,10 @@ private function preparePaymentRequest($order, $orderLines, $orderId, $terminalI
                 $request->setLanguage($langArr[0]);
             }
         }
+        $quote = $this->quote->loadByIdWithoutStore($order->getQuoteId());
+        if($this->validateQuote($quote)){
```

Comment:
Inconsistent code style
```suggestion
        if ($this->validateQuote($quote)) {
```